jujugui/static/gui/src/app/components/env-list/env-list.js

```diff
@@ -101,7 +103,10 @@ YUI.add('env-list', function() {
         <juju.components.Panel
           instanceName="env-list-panel"
           visible={true}>
-          <ul className="env-list">
+          <ul className="env-list"
+            role="menu"
+            aria-hidden="true"
```

Comment:
I don't think that this is hidden all the time - it should probably be a boolean based on the visibility of the list.